### DISCUSSION TITLE
GDBus Error

### DISCUSSION THREAD

**Discussion by iamzoltan** [gvolpe/nix-config#19] [Q&A] [answered]
Hey,

I am trying to build a slightly modified version of what you go. I feel like I am almost there, through lots of tinkering, but I am running into this issue:

   Activating dconfSettings
   error: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name ca.desrt.dconf was not 
   provided by any .service files

after I run the result/activate command

any chance you know why or would be willing to help me debug?

**Comment by gvolpe** [comment, marked as answer]
I remember seeing this one around, do you have these in your configuration?

```nix

{
  programs.dconf.enable = true;

  services = {
    dbus = {
      enable = true;
      packages = [ pkgs.gnome3.dconf ];
    };
  };
}
```
